pkg/selinux/setroubleshoot-client.js

```diff
@@ -21,6 +21,7 @@ var cockpit = require("cockpit");
 var _ = cockpit.gettext;
 
 var moment = require("moment");
+moment.locale(cockpit.language);
```

Comment:
As @allisonkarlitskaya pointed out in #9349 , we have this dichtonomy of POSIX-style locales (e. g. `pt_BR`) vs. JS-style locales (`pt-br`). Can you please check this with Brazilian Portugese? Possibly it will not work and this will need a similar translation. At that point it might justify a helper function in cockpit.js.

Reply:
Tested, seems moment.js (unlike JS's `toLocaleString()`) gets along with POSIXy locales just fine.